definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+                set -o pipefail
+                set -o errexit
+
+                name=$1
+                bam=$2
+                reference_fasta=$3  #assumes that there's a .genome file next to the reference .fa
+                genome_file = `basename $3 .fa`
+                if [[ -f "$genome_file" ]];
+                then
+                    echo "File $genome_file does exist."
+                else
+                    echo "File $genome_file does not exist. Please make sure that the .genome file is in the same directory as the .fa file."
+                    exit 1;
+                fi
+
+                export PATH=$PATH:/opt/homer/bin
+                if [[ ! -d $name/homer ]];then
+                    mkdir -p $name/homer
+                fi
+                echo "creating tagDir"
+                makeTagDirectory $name/homer $bam
```

Comment:
instead of modifying the path on line 31 why don't you write the full path `/opt/homer/bin/makeTagDirectory` here instead?

Reply:
adding to the PATH is needed here, because some of these homer tools call other tools internally and expect them to be in the PATH.